microsoft-edge/webview2/reference/javascript/hostobjectasyncproxybase.yml

```diff
@@ -125,14 +125,13 @@ methods:
       content: 'setLocalProperty(propertyName: string, propertyValue: any): any;'
       parameters:
         - id: propertyName
-          description: Name of the property to get the value of.
+          description: Name of the property of which to set the value.
           type: string
         - id: propertyValue
           description: Value to set the property to.
           type: any
       return:
         type: any
         description: >-
-          A promise representing the value of the property after it is set. This promise only resolves after the
-          property changes value.
+          The value of the property after it is set.
 extends: <a href="https://microsoft.github.io/PowerBI-JavaScript/interfaces/_node_modules_typedoc_node_modules_typescript_lib_lib_es5_d_.callablefunction.html">CallableFunction</a>

```

Comment:
404 #1
i confirmed that in the live docs (& this PR), the link "Extends CallableFunction" is 404.
in live page:
https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/javascript/hostobjectasyncproxybase
[CallableFunction] url is:
https://microsoft.github.io/PowerBI-JavaScript/interfaces/_node_modules_typedoc_node_modules_typescript_lib_lib_es5_d_.callablefunction.html
that's 404
line 138 in main branch:
https://github.com/srwei/edge-developer/blob/main/microsoft-edge/webview2/reference/javascript/hostobjectasyncproxybase.yml
= 137 in pr's branch

Reply:
en-us/ removed.  Closing comment.  Dave is researching where CallableFunciton is defined.